### DISCUSSION TITLE
Welcome to vue-pdf Discussions!

### DISCUSSION THREAD

**Discussion by teckel12** [teckel12/vue-pdf#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mercader1234** [comment]
为什么我使用vue-pdf,切换pdf预览的时候会报错找不到destroy()???
